docker/image/whyis-server/install.pp

```diff
@@ -0,0 +1,20 @@
+Exec { path => ["/usr/local/sbin","/usr/local/bin","/usr/sbin","/usr/bin","/bin"]}
```

Comment:
This file can probably just go away, since it's just copying files. The npm install should be part of the multi-stage build of whyis-server. (Doesn't need to be in -deps, we don't have that many node deps.)